### DISCUSSION TITLE
Scoop recipe

### DISCUSSION THREAD

**Discussion by melMass** [Dashlane/dashlane-cli#254] [Show and tell]
Hello,

First of all thanks a lot for these projects (cli/action) I've been a customer for years but I'm just discovering it all and it's beyond my expectations, I really love it.

I made a recipe for scoop that you are welcome to fork into your own bucket: https://github.com/melMass/scoop-mtb/commit/794ad3dfe87575241e1c309020656a412eac1eed

IMHO you should somehow advertise these more, 1Password does a lot of "marketing for dev" and I was planning on switching because of that.

Thanks

**Comment by Mikescops** [comment]
Hello,

Thanks for sharing, I'll take a look at scoop I'm not familiar with it (mostly know about Chocolatey).
My main concern is the autoupdater, I hope it works well because I can't afford to maintain integration with all existing package managers (for now we support brew).
Also we're working with Microsoft to have the CLI on the official store too.

I'll share your thoughts with the marketing team, thanks.

**Reply by melMass** [reply]
Scoop is a binary PM (technically you can build packages from source but it's not common), their updater is super simple especially for GitHub, it checks for tagged releases and match them against that regex. So as long as you keep release names as is it will work an auto update (with auto hash)

There are "community" buckets for scoop, given how simple the recipe is I can try to PR it to Main or Extras

**Reply by Mikescops** [reply]
@melMass feel free to submit what you did to their main and ping me on it. We can also add it to our install instructions on the documentation then.

**Reply by melMass** [reply]
Grrr, they have a strange policy on main that this project doesn't meet yet but I'll still try :)

![image](https://github.com/Dashlane/dashlane-cli/assets/7041726/082ef95f-93f0-4eb6-a0ce-12a69e4d1251)

**Reply by melMass** [reply]
Ok found a ~~"trick"~~ 😅 

![image](https://github.com/Dashlane/dashlane-cli/assets/7041726/b4687eef-60f2-49f8-bc6c-1d02529145cb)


https://github.com/ScoopInstaller/Main/issues/5902

**Reply by Mikescops** [reply]
Ah right we're not there, but I guess we're known enough to be in.

**Reply by Mikescops** [reply]
@melMass I have released a new version just now, can you check if the updater of scoop worked?

**Reply by melMass** [reply]
It is ran every [4 hours](https://github.com/melMass/scoop-mtb/blob/953b733843f90e695087da22f550159d4935975a/.github/workflows/excavator.yml#L4-L5) for my bucket (IIRC scoop is about the same), I'll report when it did! 
You can check all the auto-updating commits of my bucket here: https://github.com/melMass/scoop-mtb/commits/master/?author=github-actions%5Bbot%5D

**Reply by Mikescops** [reply]
@melMass it worked well nice!

**Reply by melMass** [reply]
I dispatched it since it was going to be another 2 hour 😄 ...

Here is the full run down from the user's perspective on update:


https://github.com/Dashlane/dashlane-cli/assets/7041726/262e0f2a-c61f-472c-b850-43ec7532c209

**Reply by Mikescops** [reply]
Very nice!
